pkg/services/org/orgimpl/org.go

```diff
@@ -243,3 +243,45 @@ func (s *Service) Create(ctx context.Context, cmd *org.CreateOrgCommand) (*org.O
 		Updated:  q.Result.Updated,
 	}, nil
 }
+
+// TODO: refactor
```

Comment:
which part needs refactoring? there are notes under the store impl, too; it would help to know what's going to be refactored.

Reply:
in this step I only moved the sqlstore code to org service/store, it is the same refactor, make it CRUD, I will write a better comment